lagunita/lms/static/sass/partials/base/custom_overrides/_header.scss

```diff
@@ -51,3 +51,44 @@
         }
     }
 }
+section.container {
```

Comment:
Oh, I need to move this somewhere better.

Reply:
maybe add a new `_mobile.scss`?